examples/browser/cql4browsers.js

```diff
@@ -10255,11 +10255,12 @@ var ParameterDef = /*#__PURE__*/function (_Expression) {
     value: function exec(ctx) {
       // If context parameters contains the name, return value.
       if (ctx && ctx.parameters[this.name] !== undefined) {
-        return ctx.parameters[this.name]; // If default type exists, execute the default type
+        return ctx.parameters[this.name]; // If the parent context contains the name, return that
+      } else if (ctx.getParentParameter(this.name) !== undefined) {
+        var parentParam = ctx.getParentParameter(this.name);
+        return parentParam.default != null ? parentParam.default.execute(ctx) : parentParam; // If default type exists, execute the default type
       } else if (this.default != null) {
-        return this.default != null ? this.default.execute(ctx) : undefined; // Else, if context and context's parent exist return the value of the parent's parameters with the given name.
-      } else {
-        return ctx.getParentParameter(this.name);
+        return this.default != null ? this.default.execute(ctx) : undefined;
```

Comment:
The check on `this.default != null` is unnecessary here since it's in a block that goes through a `this.default != null` gate.